src/Pages/EmailList/EmailTemplate.js

```diff
@@ -0,0 +1,304 @@
+      '<a href="http://sce.engr.sjsu.edu">http://sce.engr.sjsu.edu</a><br />' +
+      'Contact: <a href="mailto:sce.sjsu@gmail.com">sce.sjsu@gmail.com</a>' +
+      '<br />Facebook: <a href="https://www.facebook.com/sjsusce/">@scesjsu' +
+      '</a><br />Instagram: <a href="https://www.instagram.com/sjsusce/">' +
+      '@scesjsu</a><br />CmpE Slack: <a href="https://cmpesjsu.slack.com/?' +
+      'redir=%2Fmessages%2FCH6TTGXLG#/">#sce</a></span></p>';
+    let data = undefined;
+
+    try {
+      let x = 0;
+      for (x = 0; x < events.responseData.length; x++) {
+        let responseData = events.responseData[x];
+        let eventDateFormatted = this.returnProperFormattedDate(
+          responseData.eventDate.toString().substring(0, 10)
+        );
+
+        const dataSingle =
+          '<p><br /><span style="font-size: 14pt; color: #0055a2;"><strong>' +
+          responseData.title +
+          '</strong></span><br /><span style="font-size: 10pt;">' +
+          responseData.description +
+          '</span><br /><br />' +
+          '<span style="font-size: 10pt;"><strong>When:</strong>&nbsp;' +
+          eventDateFormatted +
+          '&nbsp;at&nbsp;' +
+          responseData.startTime.replace(/ /g, '') +
+          '-' +
+          responseData.endTime.replace(/ /g, '') +
+          '</span><br /><span style="font-size: 10pt;"><strong>Where:' +
+          '</strong>&nbsp;' +
+          responseData.eventLocation +
+          '</span></p>\n<p><img style="max-height: 192px;' +
+          ' max-width: 192px;" src="' +
+          responseData.imageURL +
+          '" /></p>\n';
+
+        if (x === 0) {
+          data = dataSingle;
+        } else {
+          data = data + dataSingle;
+        }
+      }
+      data =
+        mailHeader +
+        data +
+        '<p>𝔁𝓾𝓮🥶𝓱𝓾𝓪🧚𝓹𝓲𝓪𝓸😻𝓹𝓲𝓪𝓸🗿𝓫𝓮𝓲👺𝓯𝓮𝓷𝓰🤩' +
+        '𝔁𝓲𝓪𝓸😼𝔁𝓲𝓪𝓸👣</p>\n<p>&nbsp;</p>\n' +
+        mailFooter;
+    } catch (error) {
+      return error;
+    }
+    this.updateContent(data);
+  };
+
+  // Used by handleLoadContent() to format the dates from getAllEvents()
+  // to MM/DD/YYYY format
+  returnProperFormattedDate = (date) => {
+    if (date !== null) {
+      let year = date.substring(0, 4);
+      return date.substring(5, date.length) + '-' + year;
+    } else {
+      return null;
+    }
+  };
+
+  // Update 'subject' state
+  updateSubject = (e) => {
+    this.setState({ subject: e });
+  };
+
+  // Update 'recipients' state
+  updateRecipients = (e) => {
+    this.setState({ recipients: e });
+  };
+
+  // Update 'content' state. 'content' is where the template created is stored.
+  updateContent = (e) => {
+    this.setState({ content: e });
+  };
+
+  // Clear Button 'Disabled' prop boolean
+  checkEmpty = () => {
+    return (
+      this.state.loadedContent === '' &&
+      this.state.subject === '' &&
+      this.state.recipients === ''
+    );
+  };
+
+  // Send Button 'Disabled' prop boolean
+  checkEmptyInputs = () => {
```

Comment:
your code is self documenting, so u can delete a lot of these comments, up to you.

Reply:
Hmmm, yeah I'll go ahead and delete the more obvious ones